app/containers/CreateImportWallet/ImportWallet/ImportWallet.jsx

```diff
@@ -0,0 +1,139 @@
+// @flow
+import React from 'react'
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs'
+import { ROUTES } from '../../../core/constants'
+import CloseButton from '../../../components/CloseButton'
+import BackButton from '../../../components/BackButton'
+import Button from '../../../components/Button'
+import CreateImportWalletForm from '../../../components/CreateImportWalletForm'
+import FullHeightPanel from '../../../components/Panel/FullHeightPanel'
+import { loadWalletRecovery } from '../../Settings/Settings'
+import ImportIcon from '../../../assets/icons/import.svg'
+import AddIcon from '../../../assets/icons/add.svg'
+import styles from './ImportWallet.scss'
+
+type Props = {
+  // eslint-disable-next-line react/no-unused-prop-types
+  authenticated: boolean,
+  // eslint-disable-next-line react/no-unused-prop-types
+  setAccounts: (Array<Object>) => any,
+  // eslint-disable-next-line react/no-unused-prop-types
+  showSuccessNotification: Object => any,
+  // eslint-disable-next-line react/no-unused-prop-types
+  showErrorNotification: Object => any,
+}
+
+type State = {
+  tabIndex: number,
+}
+
+const IMPORT_KEY_OPTIONS = {
+  WIF: {
+    render: (props: Props) => (
```

Comment:
I think we can get rid of the `// eslint-disable-next-line react/no-unused-prop-types` above if we just define the props inline here `(props: {authenticated: boolean})`